spec/spec.md

```diff
@@ -638,7 +638,327 @@ Below is a storyboard that explains one concrete scenario using a workplace cred
 
 ## Examples
 
-Embedded or referenced non-normative examples
+The examples which follow are non-normative.
+
+Each example is consistent with the other examples in the section, but will not match any externally run system and should not be considered as supported or endorsed identity.
+
+### Identities
```

Comment:
not identiites... Identifiers, or a "key material of each entity" would be more accurate IMO

Reply:
yup, thanks!! this was the word i needed